### DISCUSSION TITLE
Source for build Project

### DISCUSSION THREAD

**Discussion by DevSelchow** [ammariqais/SkiaForUnity#35] [Q&A] [answered]
Hello, 
I am interested in the project you use to build your dlls.  Like the "libSkiaSharp.dll" etc.
I would like to add some additional dependencies that also depend on skia to my own project(Svg.Skia)
It would be awesome to have your project with all the different dependencies and build targets. 

Would you consider making it public as well?

**Comment by ammariqais** [comment, marked as answer]
Skia4Unity now supports [Svg.Skia/](https://wieslawsoltes.github.io/Svg.Skia/), please update your package to the latest release 3.0.0-pre6
